### PR TITLE
Fast-follows for MS Fabric docs

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Microsoft Fabric docs fast-follows
- Create new dbt Cloud MS Fabric connect page and link to it from https://docs.getdbt.com/docs/cloud/connect-data-platform/about-connections 
- Add new dbt Cloud MS Fabric connect page to sidebar
- Update this page https://docs.getdbt.com/docs/core/connect-data-platform/fabric-setup to say it's supported in dbt Cloud
- Add link to MS Fabric on https://docs.getdbt.com/docs/core/connect-data-platform/about-core-connections
- Update tile in verified adapters to point to new connect page
- Update quickstart
- Update release note

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Needs review from DX

Adding new pages (delete if not applicable):
- [x] Add page to `website/sidebars.js`
- [x] Provide a unique filename for the new page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/02-Nov-2023/microsoft-fabric-support-rn.md [hunk 1]
```diff
@@ -8,11 +8,13 @@ tags: [Nov-2023]
...
+Also, check out the [Quickstart for dbt Cloud and Microsoft Fabric](/guides/microsoft-fabric?step=1). The guide walks you through: 
 
 - Loading the Jaffle Shop sample data (provided by dbt Labs) into your Microsoft Fabric warehouse. 
 - Connecting dbt Cloud to Microsoft Fabric.
 - Turning a sample query into a model in your dbt project. A model in dbt is a SELECT statement.
 - Adding tests to your models.
 - Documenting your models.
-- Scheduling a job to run.
\ No newline at end of file
+- Scheduling a job to run
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/02-Nov-2023/microsoft-fabric-support-rn.md]:
```suggestion
- Scheduling a job to run.
```

File: website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+| Field | Description |
+| --- | --- |
+| **Server** | The service principal's **host** value for the Fabric test endpoint. |
+| **Port** | The port to connect to Microsoft Fabric. By default, it's 1433 for the standard SQL server port number. |
+| **Database** | The service principal's **database** value for the Fabric test endpoint. |
+| **Authentication** | Choose **Service Principal** from the dropdown. | 
+| **Tenant ID** | The service principal's **Directory (tenant) ID**. |
+| **Client ID** | The service principal's **application (client) ID id**. |
+| **Client secret** | The service principal's **client secret** (not the **client secret id**). |
```
Comment (dataders) [inline website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md:25]: can you somehow show that its either these fields, OR the AD user and password fields?

**Review by dataders — APPROVED** [review]
looks great! feel free to adress my comment in this PR or in a follow-on

File: website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md (hunk 2 above)
Comment (nghi-ly) [reply website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md:25]: great flag! it'd be great if we can add this info to the docs page. do we have username/password credentials that i can play around with? i'd be happy to add those required fields to the new connect page

File: website/docs/guides/microsoft-fabric-qs.md [hunk 3]
```diff
@@ -54,8 +54,8 @@ A public preview of Microsoft Fabric in dbt Cloud is now available!
     CREATE TABLE dbo.customers
     (
         [ID] [int],
-        [FIRST_NAME] [varchar] (8000),
-        [LAST_NAME] [varchar] (8000)
+        \[FIRST_NAME] [varchar](8000),
```
Comment (nghi-ly) [inline website/docs/guides/microsoft-fabric-qs.md:57]: the line `[FIRST_NAME] [varchar](8000),` in the code block doesn't render properly: 

Screenshot 2023-11-29 at 10 03 23 AM

to fix this, i needed to escape the `[` character for all instances in the code block

**Review by dataders — APPROVED** [review]
thanks @nghi-ly! :shipit:

File: website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+The following are the required fields for setting up a connection with a Microsoft Fabric using Azure AD service principal authentication. 
+
+| Field | Description |
+| --- | --- |
+| **Server** | The service principal's **host** value for the Fabric test endpoint. |
+| **Port** | The port to connect to Microsoft Fabric. You can use `1433` (the default), which is the standard SQL server port number. |
+| **Database** | The service principal's **database** value for the Fabric test endpoint. |
+| **Authentication** | Choose **Service Principal** from the dropdown. | 
+| **Tenant ID** | The service principal's **Directory (tenant) ID**. |
+| **Client ID** | The service principal's **application (client) ID id**. |
```
Comment (runleonarun) [inline website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md:24]:
```suggestion
| **Client ID** | The service principal's **application (client) ID**. |
```

File: website/docs/guides/microsoft-fabric-qs.md (hunk 3 above)
Comment (runleonarun) [reply website/docs/guides/microsoft-fabric-qs.md:57]: Looks good now!

File: website/docs/guides/microsoft-fabric-qs.md [hunk 5]
```diff
@@ -108,6 +108,9 @@ A public preview of Microsoft Fabric in dbt Cloud is now available!
 2. Enter a project name and click **Continue**.
```
Comment (runleonarun) [inline website/docs/guides/microsoft-fabric-qs.md:108]: Not super important, but to fix the code syntax in GitHub, you can add a space before ` ``` ` on line 101 (the code is light blue but should have syntax highlighting)

**Review by runleonarun — APPROVED** [review]
Looks great just two non blocking comments!

File: website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md (hunk 4 above)
Comment (nghi-ly) [reply website/docs/docs/cloud/connect-data-platform/connect-microsoft-fabric.md:24]: good flag! that's the actual UI label in the microsoft service principal credentials so gonna leave as is

**Pull request merged**